docs/core/integration-quality-scale/rules/iqs014.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "IQS014 - Entities use device classes where possible"
+---
+
+IQS014 is as follows:
+
+Entities use device classes where possible
+
+## Reasoning
+
+Device classes are a way to give context to an entity.
+These are used by Home Assistant for various purposes like:
+- Allowing the user to switch to another unit of measurement than what the device provides.
+- They are used for voice control to ask questions like "What is the temperature in the living room?".
+- They are used for exposing entities to cloud based ecosystems like Google Assistant and Amazon Alexa.
+- They are used to automatically set up a default set of icons for the entity.
+- They can be used to set a default name of the entity to decrease the burden on our translators.
+
+Because of these reasons, it is important to use device classes where possible.
+
+## Example implementation
+
+In the example below we have a temperature sensor that uses the device class `temperature`.
+The name of this entity will be `My device temperature`.
```

Comment:
Should we show an example where the translation is automatically provided by the device class?